src/app/components/playback/playback_widget.rs

```diff
@@ -112,7 +112,7 @@ impl PlaybackWidget {
             style_context.add_class(class);
             widget.seek_bar.set_range(0.0, duration);
             widget.seek_bar.set_value(0.0);
-            widget.track_position.set_text("0:00");
+            widget.track_position.set_text("0∶00");
```

Comment:
I'm not sure it's okay to put non-ansi symbols inside the code, especially if they can confuse people being much like other symbols. Maybe it should be better defined by symbol id or somewhere inside translations?

Reply:
Fair point! I think it's fine for now, and it might be a bit tricky since it appears both in source code and in .ui files. I'll accept the PR as is but we could clean this up eventually (it should be possible to not have these kinds of timestamps in the .ui files at all).